libs/constants/index.js

```diff
@@ -61,10 +61,39 @@ const ACTIVE_LOCALES = new Set([
   "zh-tw",
 ]);
 
+const CSP_DIRECTIVES = [
+  "default-src 'self'",
+  "script-src " +
+    [
+      "'report-sample'",
+      "'self'",
+      "https://cdn.speedcurve.com/js/lux.js",
+      "'sha256-q7cJjDqNO2e1L5UltvJ1LhvnYN7yJXgGO7b6h9xkL1o='", // LUX
+      "https://www.google-analytics.com/analytics.js",
+      "https://www.google-analytics.com/analytics_debug.js",
+      "https://polyfill.io/v3/polyfill.min.js",
+    ].join(" "),
+  // TODO: eventually drop unsafe-inline, after our move to Markdown
+  "style-src 'report-sample' 'self' 'unsafe-inline'",
+  "object-src 'none'",
+  "base-uri 'self'",
+  "connect-src 'self'",
+  "font-src 'self'",
+  "frame-src 'self' https://interactive-examples.mdn.mozilla.net https://mdn.github.io https://yari-demos.prod.mdn.mozit.cloud",
```

Comment:
Pretty sure this will break the YouTube embeds. And is that a matter of setting more than just `frame-src`?

Reply:
same as above, I hope we can find all those cases with our CSP reporting. But yeah you're probably very right

Reply:
We should add `https://mdn.mozillademos.org` or a lot of things will break.

Reply:
Oh, and `yari-demos.stage.mdn.mozit.cloud`.